solana/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "solana"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+solana-program = "1.14.10"
```

Comment:
This is already a very old version of `solana-program`.

I suggest that for our crates that track other crates we use the same version number as the crate being tracked.

So we will have `solana` version `1.14`, `1.17`, etc.

We might also need to maintain multiple version of the package for backward compatibility. Not sure what is the best way to do that in rust.